src/azure/Templates/AzureServiceClientTemplate.cshtml

```diff
@@ -43,7 +43,7 @@ bool hasClientLevelMethods = Model.MethodTemplateModels.Any( each => each.Method
 @EmptyLine
 @if (hasClientLevelMethods)
 {
-@:class @(Model.Name)(@(Model.Name)OperationsMixin):
+@:class @(Model.Name)(@(Model.Name)OperationsMixin, PipelineClient):
```

Comment:
Ah - so the way I wrote the pipeline client in azure-core was that it would be an attribute (`self._client`) rather than a parent.... happy to discuss this further as my decision here may not be the best one. However in the meantime this will not currently work with the way I wrote it.